charts/kyverno-crds/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: kyverno-crds
+version: v1.4.3-rc2
+appVersion: v1.4.2-rc2
```

Comment:
@treydock - we are ready to merge this PR, can you please sync the latest changes?

Reply:
Merged main and made these versions both `v1.4.2`. Wasn't sure if better to keep main kyverno chart version same as here or keep CRD chart version in sync with Kyverno itself since it's just a copy of CRD YAML and no extra logic to modify.

Reply:
Sorry forgot to mention, we bumped the Helm version to 2.0, and it was updated to v2.0.1 after this PR https://github.com/kyverno/kyverno/pull/2258. I guess we need to change it to v2.0.2.

Reply:
Bumped both charts to v2.0.2